impeller/entity/geometry/point_field_geometry.cc

```diff
@@ -64,7 +64,7 @@ GeometryResult PointFieldGeometry::GetPositionBuffer(
 
           Point center = points_[0];
           for (auto& vertex : circle_vertices) {
-            output[offset++] = Point(center + vertex);
+            output[offset++] = static_cast<Point>(center + vertex);
```

Comment:
wait, where is the cast where. both center and vertex are `Point`s

Reply:
Yea, I was looking at that.  Here is the error:
```
❌ Failures for clang-tidy on /Users/aaclarke/dev/engine/src/flutter/impeller/entity/geometry/point_field_geometry.cc:
/Users/aaclarke/dev/engine/src/flutter/impeller/entity/geometry/point_field_geometry.cc:67:32: error: C-style casts are discouraged; use static_cast (if needed, the cast may be redundant) [google-readability-casting,-warnings-as-errors]
   67 |             output[offset++] = Point(center + vertex);
      |                                ^~~~~
      |                                static_cast<Point>
```

I think the cast was redundant and the lint is able to convert it to what it's doing, not realize it's redundant.

Reply:
oic!

Reply:
There are cases where it did determine they are redundant for something like `bool(false)`, it being an expression must have messed that up.